### ISSUE TITLE
VoiceOver Accessibility support

### ISSUE THREAD

**Issue by markdorison** [AlanQuatermain/AQGridView#189]
Please correct me if I am off base, but the current AQGridView implementation does not seem to play nicely with VoiceOver accessibility support. I am unable to select grid items and have them spoken despite setting an accessibilityLabel property on the view.
Has anyone else worked with this issue before?

**Comment by ahalls** [comment]
I'm also faced with this issue.
